scripts/info.js

```diff
@@ -0,0 +1,24 @@
+var information = [];
+
+function Info (catagories) {
+  this.infoTitle = catagories.title; //Title of info
+  this.summery = catagories.summery; //Summery of info
+}
+
+Info.prototype.toHtml = function() {
+  var $infoChunk = $('section').clone();
```

Comment:
You're cloning all <section> tags in your document. You can avoid some bugs by adding 'template' as a class to your HTML <section> (which is a template after all). The code on line 12 where you are removing the class from $infoChunk will then serve its purpose and ensure you are only cloning the template HTML and not other extraneous <section> elements.